.github/workflows/quality.yml

```diff
@@ -54,15 +54,15 @@ jobs:
             echo "::set-output name=body::$body"
           fi
       - name: Find Comment
-        if: always()
+        if: always() && github.event_name == 'pull_request'
```

Comment:
I am still a bit confused about this `always()` thing. If there are other conditions is it still mandatory to have it?

Reply:
Yes while it means that this step will be executed even if another step fails